### ISSUE TITLE
npmInstall permission issue in xtext-buildenv container

### ISSUE THREAD

**Issue by kthoms** [eclipse/xtext-web#123]
Build fails in xtext-buildenv container:
```
> Task :org.eclipse.xtext.web:npmInstall FAILED
npm ERR! path /home/default/.npm/_logs
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall scandir
npm ERR! Error: EACCES: permission denied, scandir '/home/default/.npm/_logs'
npm ERR!  { Error: EACCES: permission denied, scandir '/home/default/.npm/_logs'
npm ERR!   stack: 'Error: EACCES: permission denied, scandir \'/home/default/.npm/_logs\'',
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'scandir',
npm ERR!   path: '/home/default/.npm/_logs' }
npm ERR! 
npm ERR! Please try running this command again as root/Administrator.
glob error { Error: EACCES: permission denied, scandir '/home/default/.npm/_logs'
  stack: 'Error: EACCES: permission denied, scandir \'/home/default/.npm/_logs\'',
  errno: -13,
  code: 'EACCES',
  syscall: 'scandir',
  path: '/home/default/.npm/_logs' }
npm ERR! path /home/default/.npm/_locks
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall mkdir
npm ERR! Error: EACCES: permission denied, mkdir '/home/default/.npm/_locks'
npm ERR!  { Error: EACCES: permission denied, mkdir '/home/default/.npm/_locks'
npm ERR!   stack: 'Error: EACCES: permission denied, mkdir \'/home/default/.npm/_locks\'',
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'mkdir',
npm ERR!   path: '/home/default/.npm/_locks' }
npm ERR! 
npm ERR! Please try running this command again as root/Administrator.

???????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????
???                  npm update check failed                  ???
???            Try running with sudo or get access            ???
???           to the local update config store via            ???
??? sudo chown -R $USER:$(id -gn $USER) /home/default/.config ???
???????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????
```
[xtext-web-build.txt](https://github.com/eclipse/xtext-web/files/3081543/xtext-web-build.txt)

**Comment by cdietrich** [comment]
Why is home not accessible?

**Comment by kthoms** [comment]
```
+ whoami
whoami: cannot find name for user ID 1000100000
```

**Comment by kthoms** [comment]
https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container

**Comment by cdietrich** [comment]
Fixed in 2.18
